parse README (#6)
broadinstitute/pooled-cp-profiling-template: setup_README.md

> The following are the two setup steps that need to be performed once at the start of a project.  

Reviewer: ```suggestion
The following are the two setup steps that need to be performed **only once** at the start of an experiment.  
```

Author: I think of each batch as possibly being a separate experiment, and these steps don't need to happen with each batch. So if you don't like "project" and I don't like "experiment", can we come up with another word?

Author: And/or we should put into the documentation how we define project/experiment/batch?

Reviewer: Want to say "start of each batch of data collection" or "start of each project batch"? Let's keep **only once** though.

I agree the data pipeline welding "unit" will be experimental batch. An experiment may contain multiple batches, and a project may contain multiple experiments (in my view). 

Although we may eventually want to make the recipe focused at the "experiment" level (as defined above) since we are likely to want to develop batch effect correction methods. These tools should be part of the recipe IMO - this is a bit down the road though

Author: I just added a commit that defines our terms and so it should be consistent now.

I agree we want to eventually bring it to the experiment level ([related issue](https://github.com/broadinstitute/pooled-cell-painting-profiling-recipe/issues/20)).

For a general overview of the pipeline welding process, see the [repo README](README.md).  
For the welding process steps to perform with each dataset, see the [weld process README](weld_process_README.md).

## Setup the Computational Environment

Install [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/).  
We use conda as an environment manager.

```bash
# Install computational environment
conda env create --force --file environment.yml

# Initialize the environment
conda activate pooled-cell-painting
```

## Fork the Pooled Cell Painting Painting Recipe

We first want to [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) the official pooled cell profiling recipe located at https://github.com/broadinstitute/pooled-cp-profiling-recipe.

* **Result:**  
The fork creates a copy of a recipe repository.  
* **Goals:**  
1) Remove the connection to official recipe updates to avoid unintended weld versioning reversal.  
2) Enable independent updates to fork code that does not impact official recipe.  
* **Execution:**  

> See [forking instructions](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) and the image below.

Reviewer: ```suggestion
### Result:

The fork creates a copy of a recipe repository.  

### Goals:

1. Remove the connection to the official recipe updates to avoid unintended weld versioning reversal.  
2. Enable independent updates to fork code that does not impact official recipe.  

### Execution:

See [forking instructions](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) and the image below.
```

Lets move these to H3 headings
